src/server/database/migrations/20161105214300-invitation.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable max-len */
+exports.up = async(r) => {
+  const indices = [
+    r.table('Project').indexCreate('tokenExpiration')
```

Comment:
@jordanh i could go either way on this. either we index on this, or on a compound key of team & this. Using a compound key means we can remove a filter from the changefeed query